### PR TITLE
Circulars text general

### PR DESCRIPTION
**Pull request by israelmcmc**
- There's a link to the Notices page that we'll have to fix when that page is created.
- Doesn't include yet @eburnsastro's text or link on the style guidelines

### CODE REVIEW

File: app/routes/circulars.md [hunk 1]
```diff
@@ -5,16 +5,123 @@ meta:
...
-- [Real-time archive of all GCN Circulars by date](https://gcn.gsfc.nasa.gov/gcn3_archive.html)
-- [GCN Circulars by object](https://gcn.gsfc.nasa.gov/selected.html)
+GCN Circulars are rapid astronomical bulletins submitted by and distributed to community members worldwide.
+
+- Circulars are **brief**, **citeable**, and **human-readable**.
+- Circulars can announce observations, quantitative near-term predictions, requests for follow-up observations, or future observing plans.
+
+Circulars allow the GRB community to **email** messages to a central queue where they are automatically vetted and distributed to the entire GRB community. These prose-style messages allow the community to coordinate and make optimum use of its limited resources. 
+
+The GCN Circulars list is independent from the machine-readable [GCN Notices (TODO:link)](notices.md) system.
```
Comment (lpsinger) [inline app/routes/circulars.md:15]:
```suggestion
The GCN Circulars list is independent from the machine-readable [GCN Notices (TODO:link)](notices) system.
```

File: app/routes/circulars.md [hunk 2]
```diff
@@ -5,16 +5,123 @@ meta:
...
+
+
+ > **Quick [submission](#circular-submission) refresher**
+ > 1. Email your submission to gcncirc@capella2.gsfc.nasa.gov.
+ > 2. The email subject-line must contain at least one of the strings in the [list of instruments and source types](#subject-word-list).
```
Comment (lpsinger) [inline app/routes/circulars.md:20]: Do you intend this to be a [summary box](https://designsystem.digital.gov/components/summary-box/)? If so, I can format that for you later.

File: app/routes/circulars.md [hunk 3]
```diff
@@ -5,16 +5,123 @@ meta:
...
+
+<a name="subscribing"></a>
```
Comment (lpsinger) [inline app/routes/circulars.md:22]: I think that you can remove the anchors. Our typesetting system will create anchors automatically.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/circulars.md (hunk 2 above)
Comment (israelmcmc) [reply app/routes/circulars.md:20]: Yes, that looks good. Thanks.

File: app/routes/circulars.md (hunk 3 above)
Comment (israelmcmc) [reply app/routes/circulars.md]: Makes sense. I just left the anchor for the mandatory words list since that's not a section

**Pull request merged**